pypeman/contrib/hl7.py

```diff
@@ -120,10 +125,10 @@ async def start(self):
                 protocol_factory=lambda: MLLPProtocol(self.handler, loop=self.loop),
                 sock=self.sock_obj,
             )
-            print("MLLP server started at http://{}:{}".format(self.address, self.port))
+            logger.debug("MLLP server started at %r", self.sock)
```

Comment:
to avoid socker logger issues with pickle:
`  logger.debug("MLLP server started at %s", repr(self.sock))`

Reply:
done